src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,114 +35,121 @@
-                // SwerveSubsystem.getInstance().setDefaultCommand();
-        }
-
-        public Command getAutonomousCommand() {
-                return autoChooser.getSelected();
-        }
-
-        public void setMotorBrake(boolean brake) {
-                SwerveSubsystem.getInstance().setMotorBrake(brake);
-        }
+    // The robot's subsystems and commands are defined here...
+    // Replace with CommandPS4Controller or CommandJoystick if needed
+    CommandJoystick driverController = new CommandJoystick(Constants.OperatorConstants.JOYSTICK_PORT);
+
+    // CommandJoystick driverController = new
+
+    private final VisionEstimation visionEstimation = new VisionEstimation();
+
+    private final SendableChooser<Command> autoChooser;
+
+    /**
+     * The container for the robot. Contains subsystems, OI devices, and commands.
+     */
+    public RobotContainer() {
+        // Configure the trigger bindings
+        configureBindings();
+        // AbsoluteDrive closedAbsoluteDrive = new
+        // AbsoluteDrive(SwerveSubsystem.getInstance(),
+        // // Applies deadbands and inverts controls because joysticks
+        // // are back-right positive while robot
+        // // controls are front-left positive
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftY(),
+        // OperatorConstants.LEFT_Y_DEADBAND),
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftX(),
+        // OperatorConstants.LEFT_X_DEADBAND),
+        // () -> -driverXbox.getRightX(),
+        // () -> -driverXbox.getRightY());
+
+        // AbsoluteFieldDrive closedFieldAbsoluteDrive = new
+        // AbsoluteFieldDrive(SwerveSubsystem.getInstance(),
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftY(),
+        // OperatorConstants.LEFT_Y_DEADBAND),
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftX(),
+        // OperatorConstants.LEFT_X_DEADBAND),
+        // () -> driverXbox.getRawAxis(2));
+
+        // AbsoluteDriveAdv closedAbsoluteDriveAdv = new
+        // AbsoluteDriveAdv(SwerveSubsystem.getInstance(),
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftY(),
+        // OperatorConstants.LEFT_Y_DEADBAND),
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftX(),
+        // OperatorConstants.LEFT_X_DEADBAND),
+        // () -> MathUtil.applyDeadband(driverXbox.getRightX(),
+        // OperatorConstants.RIGHT_X_DEADBAND),
+        // driverXbox::getYButtonPressed,
+        // driverXbox::getAButtonPressed,
+        // driverXbox::getXButtonPressed,
+        // driverXbox::getBButtonPressed);
+
+        // TeleopDrive simClosedFieldRel = new
+        // TeleopDrive(SwerveSubsystem.getInstance(),
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftY(),
+        // OperatorConstants.LEFT_Y_DEADBAND),
+        // () -> MathUtil.applyDeadband(driverXbox.getLeftX(),
+        // OperatorConstants.LEFT_X_DEADBAND),
+        // () -> driverXbox.getRawAxis(2), () -> true);
```

Comment:
Generally don't keep things commented out. Instead, figure out a way to switch between these dynamically if you _might_ need it later, or just delete it